core/core-security/src/main/java/com/pgms/coresecurity/security/jwt/booking/BookingAuthEntryPoint.java

```diff
@@ -28,11 +30,11 @@ public BookingAuthEntryPoint() {
 	@Override
 	public void commence(HttpServletRequest request, HttpServletResponse response,
 		AuthenticationException authException) throws IOException, ServletException {
```

Comment:
HttpResponseUtil 을 만들어 놓은게 있어서 이걸 적용하면 될 거 같아요~!

```suggestion

@Slf4j
public class BookingAuthEntryPoint implements AuthenticationEntryPoint {

	@Override
	public void commence(HttpServletRequest request, HttpServletResponse response,
		AuthenticationException authException) throws IOException {
		BaseErrorCode errorCode = BookingErrorCode.BOOKING_TOKEN_NOT_EXIST;
		ErrorResponse errorResponse = errorCode.getErrorResponse();
		HttpResponseUtil.setErrorResponse(response, errorCode.getStatus(), errorResponse);
	}
}

```

Reply:
제거했습니다!